tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          pypy,
+          pypy3,
+          coverage
+
+[testenv]
+usedevelop = true
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=86
```

Comment:
I keep seeing 86.  I don't believe all the packages just coincidentally currently have 86% coverage.

Reply:
You are right, I just copied `tox.ini` from zope.app.appsetup. 😁